Added refactored iOS vision task runner sources

priankakariatyml opened the pull request.

schmidt-sebastian reviewed (changes requested):
Do you mind fixing this?

```
convenience initializer should not invoke an initializer on 'super' [-Werror,-Wobjc-designated-initializers]
  100 |   self = [super initWithCalculatorGraphConfig:graphConfig
      |                 ^

```

This turns out to be more involved that I thought (probably not super complicated, but it might be better to update the PR).

priankakariatyml commented:
> convenience initializer should not invoke an initializer on 'super'



> Do you mind fixing this?
> 
> ```
> convenience initializer should not invoke an initializer on 'super' [-Werror,-Wobjc-designated-initializers]
>   100 |   self = [super initWithCalculatorGraphConfig:graphConfig
>       |                 ^
> ```
>
> This turns out to be more involved that I thought (probably not super complicated, but it might be better to update the PR).

I thought this PR was approved. Sorry, I didn't notice this. I have fixed it. Removed `initWithCalculatorGraphConfig`. We need only one designated initialiser which does everything
